Encounter Me/Pages/Registration.razor

```diff
@@ -71,7 +71,7 @@
         isRegistrationSuccess = true;
 
         await userDataService.AddUser(new UserData(
-            id: 0, // TODO: change to guid
+            id: Guid.NewGuid(), // TODO: change to guid
```

Comment:
You could have also deleted obsolete comment.

Reply:
Just deleted.